src/Distribution/Nixpkgs/Haskell/FromCabal.hs

```diff
@@ -139,6 +139,7 @@ fromPackageDescription haskellResolver nixpkgsResolver missingDeps flags Package
       | i == "gtk2"                      = binding # (i, path # ["pkgs","gnome2","gtk"])
       | i == "gtk3"                      = binding # (i, path # ["pkgs","gnome3","gtk"])
       | i == "gtksourceview3"            = binding # (i, path # ["pkgs","gnome3","gtksourceview"])
+      | i == "vte"                       = binding # (i, path # ["pkgs","gnome3","vte"])
```

Comment:
This change will break all packages that assume `vte` to come from `gnome2`. Note that, unlike the example above it, this entry has no "3" baked into its name and is therefore ambiguous.

Reply:
Good catch @peti.  I just created commit 5fab8e6 that I think will fix this.  It specifically checked for the `vte-2.91` library (instead of just any version of `vte`).

You can see that the `gi-vte` Haskell package is specifically depending on `vte-2.91` from Gnome 3:

http://hackage.haskell.org/package/gi-vte-2.91.18/src/gi-vte.cabal

However, the older `vte` Haskell package is just depending on the `vte` library from Gnome 2:

https://github.com/gtk2hs/vte/blob/edd23698501af524610153121eff15a03d9c41c3/vte.cabal-renamed#L72

With commit 5fab8e6, both the `gi-vte` and `vte` Haskell packages should get created correctly by hackage2nix.

How does this look?